### PR TITLE
housekeeping: Use StartsWith in the csproj files to make it easier to target new versions

### PR DESCRIPTION
**Pull request by martijn00**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**

Feature

**What is the current behavior? (You can also link to an open issue here)**

Say you want to target MonoAndroid81 instead of the current MonoAndroid80, you need to change all properties in the csproj file.

**What is the new behavior (if this is a feature change)?**

It is now easier because the names should only contain the platform.

**What might this PR break?**

Nothing

**Please check if the PR fulfills these requirements**
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**Other information**:

### CODE REVIEW

**Comment by ghuntley** [comment]
Aww shucks @martijn00. I was totally going to steal this from mvx and you have done it for us. Thank-you. ❤️

**Comment by martijn00** [comment]
You could even change the folder names just like Mvx to be without the version number. Say you switch from netstandard2.0 to 2.1, the code probably still works. It would also be possible to target 1.0 and 2.0 from 1 folder. I just didn't want to make this change too big.

**Comment by ghuntley** [comment]
Good call - yes changing folder names would be too big of a change right now.

**Comment by ghuntley** [comment]
@olevett I've broken something - the API approval tests are tripping. Any chance you could lend me a fresh set of eyes?

**Comment by ghuntley** [comment]
https://github.com/reactiveui/ReactiveUI/pull/1640 needs to be merged before this.

**Comment by olevett** [comment]
Will take a look...

**Comment by ghuntley** [comment]
👍 thanks Olly. This will need some additional eyeballs/changes as that `winforms` PR was merged and this PR doesn't account for it.

File: src/Directory.build.props [hunk 1]
```diff
@@ -8,7 +8,13 @@
     <Owners>xpaulbettsx;ghuntley</Owners>
     <PackageTags>mvvm;reactiveui;rx;reactive extensions;observable;LINQ;events;frp;xamarin;android;ios;mac;forms;monodroid;monotouch;xamarin.android;xamarin.ios;xamarin.forms;xamarin.mac;wpf;net;netstandard;net461;uwp;tizen</PackageTags>
     <PackageReleaseNotes>https://reactiveui.net/blog/</PackageReleaseNotes>
-
+	<PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
+	<PublishRepositoryUrl>true</PublishRepositoryUrl>
+    <RepositoryUrl>https://github.com/reactiveui/ReactiveUI</RepositoryUrl>
+    <RepositoryType>git</RepositoryType>
+    <Product>$(AssemblyName) ($(TargetFramework))</Product>
```
Comment (olevett) [inline src/Directory.build.props]: Just a thought, is this behaving here? There's another PR around it I think where AssemblyName is sometimes null?

**Comment by glennawatson** [comment]
Given we have some conflicts can we get those fixed then we can work on getting this one integrated across?

File: src/Directory.build.props (hunk 1 above)
Comment (glennawatson) [reply src/Directory.build.props]: Let the other review do this, since it's fixing it properly.

**Comment by glennawatson** [comment]
Thanks @martijn00 - Our PR list is now empty, thanks for the contribution 👍

**Pull request merged**